scripts/check_deps.sh

```diff
@@ -35,20 +35,33 @@ missing_dep() {
 }
 
 GO_DEPS=(
-    "$GO_BIN/stringer"
-    "$GO_BIN/msgp"
-    "$GO_BIN/golangci-lint"
+    "msgp"
+    "golangci-lint"
```

Comment:
`go version -m golangci-lint` doesn't work for me. 
I have to use `$GOPATH/bin/golangci-lint --version | awk '{print $4}' | head -n1` to get the version number.